playbooks/webapp/migrate_webapp.yaml

```
@@ -8,8 +8,8 @@
 
   module_defaults:
     group/aws:
-      aws_access_key: "{{ aws_access_key }}"
-      aws_secret_key: "{{ aws_secret_key }}"
+      aws_access_key: "{{ aws_access_key | default(omit) }}"
+      aws_secret_key: "{{ aws_secret_key | default(omit) }}"
       security_token: "{{ security_token | default(omit) }}"
```

```suggestion
      session_token: "{{ security_token | default(omit) }}"
      profile: "{{ aws_profile | default(omit) }}"
```
`security_token` has been deprecated and will be removed after 2024-12-01. adding support for `aws_profile` in case user do not want to connect using credentials